### ISSUE TITLE
.remove returns element

### ISSUE THREAD

**Issue by mafintosh** [chriso/lru#15]
Currently `lru.remove(key)` returns the element and not the value. I think it would be more consistent if just returned the value like set/get.

**Comment by chriso** [comment]
Agreed. I'd be happy to change this and then bump the major to 2.0.0.

**Comment by bsonntag** [comment]
Also, the readme says that the element returned by `.remove` has a `key` property, but it doesn't:

```
var cache = LRU(3);
cache.set('foo', 'bar');
cache.remove('foo'); // => { value: 'bar', next: null, prev: null }
```

**Comment by chriso** [comment]
Fixed by #17.
